Dockerfile

```diff
@@ -3,4 +3,6 @@ RUN apk add ca-certificates
 
 FROM builder
 WORKDIR /app
-COPY k8stream agent
+
+COPY deploy/run.sh run.sh
+RUN chmod a+x run.sh
```

Comment:
Move the container to last9 repo. This container does not build k8stream and should not be here.